src/query/service/src/interpreters/interpreter_table_drop_column.rs

```diff
@@ -133,6 +137,26 @@ impl Interpreter for DropTableColumnInterpreter {
         )
         .await?;
 
+        let field = table
+            .get_table_info()
+            .meta
+            .schema
+            .field_with_name(self.plan.column.as_str())?;
+        if field.auto_increment_display.is_some() {
+            let auto_increment_name =
+                AutoIncrement::sequence_name(table.get_id(), field.column_id());
+
+            DropSequenceInterpreter::req_execute(
```

Comment:
This req_execute maybe also make a same transation into commit_table_meta?

Drop a column that bound a sequence we also need to drop the sequence?